incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -229,23 +230,60 @@ func RunCommand(cmdln ...string) (string, error) {
 	stdout, err := cmd.CombinedOutput()
 	return string(stdout), err
 }
+// CreateNamespace creates the specified namespace with canned testing labels making it easier
+// to look up and delete later.
+func CreateNamespace( ns string)   {
+	createNamespaceWithLabels(ns,"",eventuallyTimeout)
+}
+// CreateSubnamespace creates the specified namespace in the parent namespace with canned testing labels making it easier
+// to look up and delete later.
+func CreateSubnamespace( ns string,parent string)   {
+	createNamespaceWithLabels(ns,parent,eventuallyTimeout)
+}
 
-// CleanupNamespaces does everything it can to delete the passed-in namespaces. It also uses very
+// CreateNamespaceWithLabels creates a namespace or a subnamespace (if a parent is specified) and marks the namespace
+// with all the labels specified
+func createNamespaceWithLabels( ns string, parent string,timeout float64)  {
+	if parent == ""{
+		mustRunWithTimeout(1,timeout,"kubectl create ns", ns)
+	}else{
+		mustRunWithTimeout(1,timeout, "kubectl hns create", ns, "-n", parent)
+	}
+	MustRun("kubectl label --overwrite ns", ns,testingNamespaceLabel+"=true")
+}
+// CleanupTestNamespaces finds the list of namespaces labeled as test namespaces and delegates
+// to cleanupNamespaces function.
+func CleanupTestNamespaces(){
+	labelQuery := testingNamespaceLabel+"=true"
+	EventuallyWithOffset(1, func() string {
+		out,err:=RunCommand("kubectl get ns -o custom-columns=:.metadata.name --no-headers=true","-l",labelQuery)
```

Comment:
Did you run `gofmt` after this line? Or run `make fmt` from the HNC root directory. Our coding standard is to have a space after every comma.

Reply:
I ran go fmt before the commit. Will check again.